src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Constants/LoggingKeys.cs

```diff
@@ -1,32 +1,101 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using Microsoft.Extensions.Logging;
+
 namespace Microsoft.Azure.WebJobs.Host.Loggers
 {
-    internal static class LoggingKeys
+    /// <summary>
+    /// Keys used by the <see cref="ILogger"/> infrastructure.
+    /// </summary>
+    public static class LoggingKeys
     {
```

Comment:
Had to make these public. These could be useful to other ILoggers as well that plug into us. 

However, I could also just share the code file between the two and keep these internal. Thoughts?

FYI -- I'll fill in the descriptions.